packages/calcite-components/src/components/block-section/block-section.tsx

```diff
@@ -199,21 +211,35 @@ export class BlockSection implements LocalizedComponent, T9nComponent, LoadableC
   }
 
   render(): VNode {
-    const { el, messages, open, text, toggleDisplay } = this;
-    const dir = getElementDir(el);
-    const arrowIcon = open
-      ? ICONS.menuOpen
-      : dir === "rtl"
-        ? ICONS.menuClosedLeft
-        : ICONS.menuClosedRight;
+    const { messages, open, text, toggleDisplay, iconFlipRtl } = this;
+    const arrowIcon = open ? ICONS.menuOpen : ICONS.menuClosed;
+
+    const iconStartEl = this.iconStart ? (
+      <calcite-icon
+        class={CSS.iconStart}
+        flipRtl={iconFlipRtl === "both" || iconFlipRtl === "start"}
+        icon={this.iconStart}
+        key="icon-start"
+        scale="s"
```

Comment:
Does this need to use the [`getIconScale`](https://github.com/Esri/calcite-design-system/blob/f1ff5e1acd0e802f6c5b635a77dedf8138d7e753/packages/calcite-components/src/utils/component.ts#L4-L6) util?

Reply:
`block/block-section` don't have `scale` currently. It'll be reviewed in https://github.com/Esri/calcite-design-system/issues/7082. As is, we think of the components that don't have scale as `medium` and use `small` icons for them.